### ISSUE TITLE
The start message is not sent 

### ISSUE THREAD

**Issue by golubvladimir** [nksmnf/nestjs-telegraf#993]
I created small bot, but I have problem with start message

My example - https://github.com/golubvladimir/nestjs-telegraf-bot

When I use only telegraf in main.ts, it works.

```
import { Telegraf } from 'telegraf';

async function bootstrap() {
  const bot = new Telegraf('***');
  bot.start((ctx) => ctx.reply('Welcome'));
  bot.help((ctx) => ctx.reply('Send me a sticker'));
  bot.launch();
}
bootstrap();
```

**Comment by UsVfxArtia** [comment]
same

**Comment by zokeoner** [comment]
Try installing version telegraf@4.10.0

**Comment by skyArony** [comment]
same

"nestjs-telegraf": "^2.6.2"
"telegraf": "^4.11.2"

**Comment by UsVfxArtia** [comment]
up

**Comment by gianda** [comment]
It works with version telegraf@4.10.0, not with ^4.11

**Comment by skyArony** [comment]
> same
> 
> "nestjs-telegraf": "^2.6.2" "telegraf": "^4.11.2"

"telegraf": "4.10.0",
"nestjs-telegraf": "^2.6.2",

It works

**Comment by NikyOFF** [comment]
Maybe https://github.com/hypeertech/nestjs-telegraf/issues/1004 can help you

**Comment by UsVfxArtia** [comment]
"nestjs-telegraf": "^2.6.3",
    "telegraf": "^4.10.0",
Works, thanks guys!

**Comment by IcanPressF** [comment]
1 Day was working, now got this error..................

`C:\Program Files\Aeo\Backend\node_modules\telegraf\lib\composer.js:487
    return triggers.map((trigger) => {
                    ^
Error: Invalid trigger
    at C:\Program Files\Aeo\Backend\node_modules\telegraf\lib\composer.js:489:19
    at Array.map (<anonymous>)
    at normalizeTriggers (C:\Program Files\Aeo\Backend\node_modules\telegraf\lib\composer.js:487:21)
    at Function.hears (C:\Program Files\Aeo\Backend\node_modules\telegraf\lib\composer.js:345:51)
    at Telegraf.hears (C:\Program Files\Aeo\Backend\node_modules\telegraf\lib\composer.js:41:34)
    at ListenersExplorerService.registerIfListener (C:\Program Files\Aeo\Backend\node_modules\nestjs-telegraf\dist\services\listeners-explorer.service.js:133:29)
    at MapIterator.iteratee (C:\Program Files\Aeo\Backend\node_modules\nestjs-telegraf\dist\services\listeners-explorer.service.js:92:84)
    at MapIterator.next (C:\Program Files\Aeo\Backend\node_modules\iterare\src\map.ts:9:39)
    at FilterIterator.next (C:\Program Files\Aeo\Backend\node_modules\iterare\src\filter.ts:11:34)
    at IteratorWithOperators.next (C:\Program Files\Aeo\Backend\node_modules\iterare\src\iterate.ts:19:28)
    at Function.from (<anonymous>)
    at IteratorWithOperators.toArray (C:\Program Files\Aeo\Backend\node_modules\iterare\src\iterate.ts:227:22)
    at MetadataScanner.scanFromPrototype (C:\Program Files\Aeo\Backend\node_modules\@nestjs\core\metadata-scanner.js:12:14)
    at ListenersExplorerService.registerListeners (C:\Program Files\Aeo\Backend\node_modules\nestjs-telegraf\dist\services\listeners-explorer.service.js:92:30)
    at C:\Program Files\Aeo\Backend\node_modules\nestjs-telegraf\dist\services\listeners-explorer.service.js:53:43
    at Array.forEach (<anonymous>)
`

**Comment by evilsprut** [comment]
Fixed in [v2.6.3](https://github.com/evilsprut/nestjs-telegraf/releases/tag/v2.6.3)
